src/dataset-builder/ConceptSearch.ts

```diff
@@ -85,57 +82,22 @@ export const ConceptSearch = (props: ConceptSearchProps) => {
         }),
       ]),
       concepts.status === 'Ready'
-        ? h(SimpleTable, {
-            'aria-label': 'concept search results',
-            underRowKey: 'underRow',
-            rowStyle: {
-              backgroundColor: 'white',
-              ...tableLeftPadding,
-            },
-            headerRowStyle: {
-              ...tableHeaderStyle,
-              ...tableLeftPadding,
-              marginTop: '1rem',
-            },
-            cellStyle: {
-              paddingTop: 10,
-              paddingBottom: 10,
-            },
+        ? h(StyledSimpleTable, {
```

Comment:
I think this change is incorrect, as `ConceptSearch` and `ConceptSelector` are used by the add criteria UI. Instead of this approach, I would use `ConceptSelector` as-is but change the model passed to it to use `hasChildren`: `false` so the nodes aren't expandable.